lib/packwerk/cli.rb

```diff
@@ -156,12 +157,24 @@ def list_validation_errors(result)
     end
 
     def parse_run(paths)
```

Comment:
I wonder if we should rename "paths" to "params" or something like that?

Reply:
Yep!